### PR TITLE
Feature: Added Slots to Order Show Page in Hub

### PR DESCRIPTION
**Pull request by przper**
Added custom Slots in Admin Hub to OrderShow Page.

### CODE REVIEW

File: packages/admin/resources/views/livewire/components/orders/show.blade.php [hunk 1]
```diff
@@ -161,6 +167,12 @@ class="flex-shrink-0 px-4 py-2 ml-4 text-xs font-bold text-gray-700 border round
           @endforeach
         </dl>
       </section>
+
+      @foreach($this->getSlotsByPosition('bottom') as $slot)
+       <div id="{{ $slot->handle }}">
+        <div>@livewire($slot->component, ['slotModel' => $order], key("top-slot-{{ $slot->handle }}"))</div>
```
Comment (ryanmitchell) [inline packages/admin/resources/views/livewire/components/orders/show.blade.php:173]: Key should be bottom-slot-xxx not top-slot-xxx

Comment (przper) [reply packages/admin/resources/views/livewire/components/orders/show.blade.php]: Fixed in https://github.com/getcandy/getcandy/pull/269/commits/d2cee7b80a9f7e52b0e350018f7e6a53209a1cba

**Comment by alecritson** [comment]
Sorry for the delay on this one @przper I think it would also be handy to have a slot for each order line and also one on the side, under the addresses and meta. What do you think?

**Comment by przper** [comment]
> Sorry for the delay on this one @przper I think it would also be handy to have a slot for each order line and also one on the side, under the addresses and meta. What do you think?

- In my use case I don't need Slots for Lines. I use slot to re-dispatch Laravel Job which tries to send XML file and sends email to customer upon success, then updates Order Status. I do not modify Order data except status.
I think adding Slots to OrderLines would be an overkill. Modifying an OrderLine would likely lead to change in Order total amount or other important param. This would require Order reevaluation. Maybe in future someone will have a good reason for OrderLine slots.
- I have added slots on top of details section and below meta section. I think you've missed that :)

**Comment by alecritson** [comment]
@glennjacobs This one is a bit overdue, you think we can get this one in?

**Comment by glennjacobs** [comment]
> @glennjacobs This one is a bit overdue, you think we can get this one in?

Yep, don't see why not.

**Review by alecritson — APPROVED** [review]

**Pull request merged**